atlasdb-dbkvs/src/main/java/com/palantir/atlasdb/keyvalue/dbkvs/impl/OracleDbTableFactory.java

```diff
@@ -69,7 +66,7 @@ public DbReadTable createRead(TableReference tableRef, ConnectionSupplier conns)
             default:
                 throw new EnumConstantNotPresentException(TableSize.class, tableSize.name());
         }
-        return new UnbatchedDbReadTable(conns, queryFactory);
+        return new BatchedDbReadTable(conns, queryFactory, exec, config);
```

Comment:
There are no usages of ``UnbatchedDbReadTable`` now, consider getting rid of it?